tests/acceptance/features/apiGraph/getUser.feature

```diff
@@ -15,76 +15,468 @@ Feature: get users
   Scenario: admin user gets the information of a user
     When user "Alice" gets information of user "Brian" using Graph API
     Then the HTTP status code should be "200"
-    And the user retrieve API response should contain the following information:
-      | displayName  | id        | mail              | onPremisesSamAccountName | accountEnabled |
-      | Brian Murphy | %uuid_v4% | brian@example.org | Brian                    | true           |
+    And the JSON data of the response should contain display name "Brian Murphy" and match
+    """
+    {
+      "type": "object",
+      "required": [
+        "displayName",
+        "id",
+        "mail",
+        "onPremisesSamAccountName",
+        "accountEnabled"
+      ],
+      "properties": {
+        "displayName": {
+          "type": "string",
+          "enum": ["Brian Murphy"]
+        },
+        "id" : {
+          "type": "string",
+          "pattern": "^%user_id_pattern%$"
```

Comment:
```suggestion
          "pattern": "^%uuid_v4%$"
```
if we use this user_id_pattern provide  more readibility and  uuid_v4 provide similarity

Reply:
I am ok with `user_id_pattern`. Because we are checking the user id's pattern which could be of any pattern (and in oCIS we have `uuidv4` for user ids)

Reply:
yes its actually the pattern we want to check and not the real `user_id`.